clang/www/cxx_dr_status.html

```diff
@@ -15598,7 +15598,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
   </tr>
   <tr class="open" id="2632">
     <td><a href="https://cplusplus.github.io/CWG/issues/2632.html">2632</a></td>
-    <td>review</td>
+    <td>drafting</td>
```

Comment:
Update you branch with the latest `main` changes, and those unrelated changes will go away.